src/drivers/intel/ssp/ssp.c

```diff
@@ -613,35 +613,49 @@ static int ssp_set_config(struct dai *dai, struct ipc_config_dai *common_config,
 	return ret;
 }
 
-/*
- * Portion of the SSP configuration should be applied just before the
- * SSP dai is activated, for either power saving or params runtime
- * configurable flexibility.
- */
-static int ssp_pre_start(struct dai *dai)
+static int ssp_set_mclk(struct dai *dai)
```

Comment:
nit-pick: maybe use the Linux working for ssp_mclk_prepare_enable and ssp_mclk_disable_unprepare?

Set and release are not clear to me.